src/com/georgidinov/util/MyMessages.java

```diff
@@ -0,0 +1,33 @@
+package com.georgidinov.util;
+
+public final class MyMessages {
```

Comment:
Not all of the constants here are related to messages, the constants which are used only once or by only 1 class, just make them constants in the class they're used in or use them as literals.